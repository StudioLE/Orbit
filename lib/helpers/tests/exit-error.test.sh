#!/bin/bash
set -uo pipefail

# ARGS
ACTUAL_FILE=$1

source /srv/lib/helpers/exit-error.sh

declare OUTPUT=""

set +e
OUTPUT+=$(exit-error "An error has occurred." 99 2>&1)
EXIT_CODE=$?
set -e
OUTPUT+=$'\n'
OUTPUT+="EXIT_CODE: ${EXIT_CODE}"

echo "${OUTPUT}" > "${ACTUAL_FILE}"
