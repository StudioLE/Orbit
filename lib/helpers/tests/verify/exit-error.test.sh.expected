❗  An error has occurred.
EXIT_CODE: 99
